tutorials/pages/create-nft-with-iota-sdk.md

```diff
@@ -0,0 +1,220 @@
+# Create an NFT with the IOTA SDK and IPFS in NodeJs
+
+In this tutorial, you will upload your desired image to [IPFS](https://ipfs.tech/) and create an NFT from the unique IPFS URL on the Shimmer testnet. This tutorial will build on the tutorial "[Setup testnet address and receive funds from the faucet](https://wiki.iota.org/tutorials/wallet-setup)". If you haven't completed it, please do so, and afterward, come back here, or make sure you have a prefunded wallet. You will use the same folder and development environment as in the mentioned prerequired tutorial, allowing you to reuse the Stronghold account with funds on it.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) v16 and above supported. Tested till v20.
+- [IOTA SDK](https://www.npmjs.com/package/@iota/sdk) v1.0.6 used here.
+- Finish the [Set up testnet address and receive funds from faucet](https://wiki.iota.org/tutorials/wallet-setup) tutorial.
+
+## Set Up
+
+### Prepare Your Development Environment
+
+Navigate to the folder you created when completing the [set up a testnet address and receive funds from the faucet tutorial](https://wiki.iota.org/tutorials/wallet-setup):
+
+```bash
+cd wallet-setup
+```
+
+#### Install Packages
+
+Next to the existing dependencies of the wallet setup tutorial, you will only need to add two more packages by running the following commands:
+
+```bash
+npm install @iota/sdk
+npm install ipfs-core
+```
+
+Afterward, your `package.json` file should contain the following dependencies:
+
+```json {6-12}
+{
+  "name": "wallet-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "create-mnemonic.js",
+  "dependencies": {
+    "@iota/sdk": "^1.0.0",
+    "dotenv": "^16.0.1",
+    "ipfs-core": "^0.15.4"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+
+---
+
+### Prepare an Image for the NFT
+
+Add your desired `*.jpg` file to the `wallet-setup` folder you created and rename it to `nft-image.jpg`.
+
+For this tutorial we'll use the following image:
+
+![NFT Image](/nft-image.jpg)
```

Comment:
Shouldn't this image be in the PR?

Reply:
Its there in static already. Used an older available image.